### ISSUE TITLE
Order price shown incorrectly in the Market UI

### ISSUE THREAD

**Issue by MattSkala** [Tribler/tribler#4131]
##### Tribler version/branch+revision: #####
7.2.0-exp1

##### Operating system and version: #####
macOS 10.14.1

##### Steps to reproduce the behavior: #####
1. Go to the Market screen
2. Click on the "Buy BTC for MB" button
3. Fill in quantity: 1 BTC, price per unit: 100 MB/BTC

##### Expected behavior: #####
An order with the volume of 1 BTC and the price of 100 MB/BTC should be created.

##### Actual behavior: #####
An order with the volume of 1 BTC and the price of 0.01 MB/BTC is added to the list, the same price is shown in the order detail. There is only an issue with the price displayed in the UI. The bid itself is created with the correct amount, as one can verify via the API:

```
{
    "trader_id": "131dac16fa36cb3c9ff1d8382a4c3a862852da49",
    "timestamp": 1545497508.527097,
    "block_hash": "8a95ce69dfe0c3e1fafa8ae0b0e99325b20c26c6e5bfaff2e141b43d99ee7a11",
    "timeout": 3600,
    "traded": 0,
    "order_number": 11,
    "assets": {
        "second": {
            "amount": 100,
            "type": "MB"
        },
        "first": {
            "amount": 100000000,
            "type": "BTC"
        }
    }
}
```

**Comment by devos50** [comment]
The price in the user interface is displayed in the `MB/BTC` asset pair. It might make more sense to reverse this and make it the same as the user has entered it.

Thanks for reporting!

**Comment by devos50** [comment]
Fixed by #4132.
